core-web/libs/edit-content/src/lib/enums/dot-edit-content-field.enum.ts

```diff
@@ -0,0 +1,6 @@
+export enum DotEditContentFieldDataType {
+    //TODO: Doc all posibles values of DataType fields
+    BOOL = 'BOOL',
+    INTEGER = 'INTEGER',
+    FLOAT = 'FLOAT'
+}
```

Comment:
I think we should separate the types a field can recieve by it's own field. Like the text can be: Text, integer or float. But Select and Radio takes all of those and Boolean as value. So both of them should have separated enums